storage/tool/pom.xml

```diff
@@ -11,6 +11,10 @@
 
     <artifactId>storage-tool</artifactId>
 
+    <properties>
+        <slf4j.version>1.7.12</slf4j.version>
```

Comment:
this should be in root pom.xml "dependencyManagement" . We have already defined this https://github.com/hortonworks/streamline/blob/master/pom.xml#L55